### PR TITLE
Don't process PyPI version list if user specifies a version.

### PR DESCRIPTION
**Pull request by gordonmessmer**

### CODE REVIEW

File: pyp2rpm/package_getters.py [hunk 1]
```diff
@@ -127,25 +127,28 @@ def __init__(self, client, name, version=None, prerelease=False,
...
         self.name = name
-        self.versions = self.client.package_releases(self.name, True)
-
-        # Use only stable versions, unless --pre was specified
-        if not prerelease:
-            self.versions = [candidate for candidate in self.versions
-                             if not parse_version(candidate).is_prerelease]
+        if version:
+            # if version is specified, will check if such version exists
+            if self.client.release_urls(name, version) == []:
```
Comment (evgeni) [inline pyp2rpm/package_getters.py]: I think `if not self.client.release_urls(…):` is more readable (as empty lists are False-y).

**Review by evgeni — APPROVED** [review]
Reading the code it should fix my issue!

**Comment by gordonmessmer** [comment]
@decathorpe

**Comment by decathorpe** [comment]
Makes sense. If it works, :shipit:

**Pull request merged**